python/postprocessing/modules/jme/jetmetUncertainties.py

```diff
@@ -366,8 +407,15 @@ def resolution_matching(jet, genjet):
                 jets_mass_nom    .append(jet_mass_nom)
 
             if not self.isData:
-              jet_pt_jerUp         = jet_pt_jerUpVal  *jet_pt
-              jet_pt_jerDown       = jet_pt_jerDownVal*jet_pt
+              jet_pt_jerUp = { jerID: jet_pt_nom for jerID in self.splitJERIDs }
+              jet_pt_jerDown = { jerID: jet_pt_nom for jerID in self.splitJERIDs }
+              jet_mass_jerUp = { jerID: jet_mass_nom for jerID in self.splitJERIDs }
```

Comment:
This line crashes for me because `jet_mass_nom` is only defined for `iJet < nJet`, i.e. only for high-pt jets, but the overall loop also runs over low-pt jets. Suggest to either replace by `jet_pt_jerNomVal*jet_mass if self.applySmearing else jet_mass` from L404, or simply move the `jet_mass_nom` definition out of the if block.

Reply:
I think this is actually tricky to reproduce because it will only happen if the first event has `nJet==0`. In other cases, there is no crash, you just get reuse the `jet_mass_nom` value from the previous iteration of the loop, which is worse than crashing

Reply:
Good catch, should be fixed now.